src/adapters/symbiosis/index.ts

```diff
@@ -124,6 +39,15 @@ const adapter: BridgeAdapter = {
   bsc: constructParams("bsc"),
   avalanche: constructParams("avax"),
   polygon: constructParams("polygon"),
+  telos: constructParams("telos"),
+  kava: constructParams("kava"),
+  boba: constructParams("boba"),
+  // boba_bnb: constructParams("boba_bnb"),
+  // zksync: constructParams("zksync"),
```

Comment:
```suggestion
  // zksync: constructParams("era"),
```
and also change config file chainMapping
because it named like that in sdk